### PR TITLE
Add offline paid subcription for Standard Notes

### PR DESCRIPTION
**Pull request by d4rklynk**
You need to pay an offline subscription while self-hosting

### CODE REVIEW

File: docs/notebooks.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ Since January 2023, Joplin supports biometrics app lock for [Android](https://jo
         - [:simple-apple: macOS](https://standardnotes.com)
         - [:simple-linux: Linux](https://standardnotes.com)
         - [:octicons-globe-16: Web](https://app.standardnotes.com/)
+        
+You still need to pay an [offline subscription](https://standardnotes.com/help/self-hosting/subscriptions) to be able to use premium features for self-hosting. The payement is done by paying monthly or yearly. The prices are really different, monthly is 5.85$/month (meaning 70.2$/year) and yearly is 3.25$/month (meaning 39$/year).
```
Comment (mfwmyfacewhen) [inline docs/notebooks.md:66]:
```suggestion
You still need to pay an [offline subscription](https://standardnotes.com/help/self-hosting/subscriptions) to be able to use premium features for self-hosting.
```

**Review by mfwmyfacewhen — APPROVED** [review]

File: docs/notebooks.md (hunk 1 above)
Comment (mfwmyfacewhen) [reply docs/notebooks.md:66]: No reason to list the specific prices/pricing model imo since those can change and not everyone uses USD.

Comment (d4rklynk) [reply docs/notebooks.md]: I just thought it was important since the prices are completely different between monthly and yearly.

**Comment by freddy-m** [comment]
> Standard Notes requires an [offline subscription](https://standardnotes.com/help/self-hosting/subscriptions) for self-hosting.

This seems friendlier, and more to-the-point, to me...

**Review by jonaharagon — CHANGES_REQUESTED** [review]
This isn't strictly true, because you can install community extensions that replicate premium features, e.g. https://github.com/jonhadfield/awesome-standard-notes

I would rather not mention this one way or the other personally, I don't see it being super relevant here.

**Comment by mfwmyfacewhen** [comment]
> This isn't strictly true, because you can install community extensions that replicate premium features, e.g. https://github.com/jonhadfield/awesome-standard-notes
> 
> I would rather not mention this one way or the other personally, I don't see it being super relevant here.

Yeah fair, I think I'll just close this then.

**Pull request closed without merging**